bin/setup

```diff
@@ -21,6 +21,7 @@ chdir APP_ROOT do
   # Install JavaScript dependencies if using Yarn
   # system('bin/yarn')
 
+
```

Comment:
I don't know why this changed but it auto generated when I updated the `bin/` folder. In any case I don't think the change will do any harm.